README.md

```diff
@@ -75,7 +75,7 @@ with:
 
 gem_home can also be installed with [homebrew]:
 
-    curl https://github.com/postmodern/gem_home/blob/master/homebrew/gem_home.rb
+    wget https://raw.github.com/postmodern/gem_home/master/homebrew/gem_home.rb
```

Comment:
I believe macOS still ships with `curl` by default, not `wget` (unlike GNU/Linux systems). Should probably change it to `curl -O`.